plugins/connectrpc/es/v0.13.1/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "plugins-connectrpc-es",
+  "version": "1.0.0",
+  "dependencies": {
+    "@connectrpc/protoc-gen-connect-es": "0.13.1"
+  }
```

Comment:
We should also check in a package-lock.json file in this directory so we get more reproducible builds.